code/datums/brain_damage/imaginary_friend.dm

```diff
@@ -0,0 +1,131 @@
+/datum/brain_trauma/special/imaginary_friend
+	name = "Imaginary Friend"
+	desc = "Patient can see and hear an imaginary person."
+	scan_desc = "partial schizophrenia"
```

Comment:
Hardly related to schizophrenia. DID seems more likely, or if you're going to complain that players wouldn't know what that is, say it's multiple personality disorder.

Reply:
Schizophrenia includes hearing voices that do not exist, so partial schizo makes sense to me.